src/lis3dh-motion-detection.cpp

```diff
@@ -345,33 +403,48 @@ void LIS3DH::applySettings( void )
 status_t LIS3DH::intConf(interrupt_t interrupt,
 						event_t moveType, 
 						uint8_t threshold,
-						uint8_t timeDur)
+						uint8_t timeDur,
+						bool polarity)
 {
 	uint8_t dataToWrite = 0;  //Temporary variable
 	status_t returnError = IMU_SUCCESS;
 
+	if ( interrupt == INT_0 ) { // disable INTerrupts and exit
+		writeRegister(LIS3DH_CTRL_REG3, 0x00);
+		returnError = writeRegister(LIS3DH_CTRL_REG6, 0x00);
+		return returnError;
+	}
+
 	uint8_t regToWrite = 0;
-	regToWrite = (interrupt == INT_1) ? LIS3DH_INT1_CFG : LIS3DH_INT2_CFG;
+	regToWrite = (interrupt == INT_1) ? LIS3DH_INT1_CFG : LIS3DH_INT2_CFG; // 0x30 or 0x34
 
 	//Build INT_CFG 0x30 or 0x34
 	//Detect movement or stop
-	if(moveType == 1)	dataToWrite |= 0x0A;
-	else 							dataToWrite |= 0x05;
+	//bike straight  : ZL YH XL
+	//bike fall right: ZL YH XH
+	//bile fall left : ZL YL XH
+	if(moveType == 0)	dataToWrite |= 0x0A; // b1010 phone on table
+	if(moveType == 1)	dataToWrite |= 0x05; // b0101 phone upside down on table
+	if(moveType == 2)	dataToWrite = 0b00000011; // bike fall
```

Comment:
This does not make sense, can you please elaborate?

XHIE Enable interrupt generation on X high event or on Direction recognition.
XLIE Enable interrupt generation on X low event or on Direction recognition.

Please remove the comments which are only relevant to your bike application.

Reply:
Yes, mistake :)

I don’t think comments where annoying. As you wish.